docs/INSTALLATION.md

```diff
@@ -0,0 +1,89 @@
+
+## Network connection
+Connect an RJ45 cable to the Ethernet port on the main unit. Connect the other end to a port on your main network (likeliy on your router). The unit's IP address is configured by DHCP by default.
+
+## Power
+**WARNING!! VERIFY THAT THE VOLTAGE OF YOUR WALL OUTLET MATCHES THE VOLTAGE LISTED NEAR THE POWER INLET.**
+
+AmpliPro ships preconfigured for the typical mains voltage in your region, either 120V mains power or 230V mains; you can see what mode it was set to before shipment based on which box is checked near the power inlet on the unit EDIT EDIT EDIT. If your unit is preconfigured incorrectly for your region, several changes need to be made to the unit to connect to alternative mains:
+- Use an appropriate voltage power cable (AmpliPro only ships with the cable rated for the region)
+- Change the internal power supply fuse located in the IEC socket (located just above the power inlet on the exterior of the AmpliPro)
+- Switch the internal power supply as needed (See below)
+
+Internally, there is a red switch on the side of the power supply that can be toggled for 230V. The easiest way to get at this switch is to take off the top cover. The power supply switch can potentially be accessed through the vent slots on the right side of the unit as well. Here are some pictures to help clarify the process:
+
+![Power Supply Configuration](imgs/manual/power_instructions.png)
+
+Once the unit has the correct input voltage configured, plug the AmpliPro in to a wall outlet. The ON/STANDBY will start of blinking red then transition to solid red once the unit is fully powered on. Continue below to enjoy your unit!
+
+## Startup and Configuration
+Now that the AmpliPro unit is powered on, it is time to play with it. Let's see what it can do:
+
+1. Go to amplipi.local (Android and Windows 7 users will need to type the ip address found on the unit's display into their web browser to find the page).
+2. You should now be connected to AmpliPro's mobile-friendly website. Please note that an https connection to the AmpliPro is not currently available since certificates have to be managed on a per-unit basis.
+3. Android users can download the AmpliPi app in the Google Play Store.
+4. Click the plus (+) icon and select a stream
+    - The Groove Salad - InternetRadio stream comes preconfigured (needs an internet connection).
+5. Change the volume on the zone you would like to output music on. Many of the zones will be hidden inside a group. Click on the different groups to see which zones belong to them.
+6. You will probably want to change the default group and zone names and add different streaming sources. The gear icon (⚙) on the bottom right leads to the settings page where you can configure inputs, zones, and groups.
+
+Since this was printed we may have added more features and made the AmpliPro better; this guide is continued at amplipi.com/getting-started. Use the QR code below to jump there now.
```

Comment:
Different wording I think sounds better:
```suggestion
Since AmpliPro is constantly improving, our manual may have been updated. Go to amplipi.com/getting-started or scan the QR code below to view the latest version of our manual.
```
Also that link is broken so I'm just assuming it's the manual